### DISCUSSION TITLE
Было бы круто иметь возможность задать песне тег

### DISCUSSION THREAD

**Discussion by NikitaVisitor** [Chimildic/YaMuTools#87] [Идея]
Было бы круто иметь возможность задавать свои теги для песен и генерировать плейлисты на основании комбинации этих тегов.

**Comment by kisoid** [comment]
Создай плейлист-тег. Название можно с # писать, чтобы аутентично было :)

Добавляй/удаляй мелодии в плейлисты-теги. Генерируй плейлисты из плейлистов... И т.д.
